src/platforms/javascript/common/troubleshooting/index.mdx

```diff
@@ -54,6 +54,13 @@ Most community CDNs properly set an `Access-Control-Allow-Origin` header.
  Access-Control-Allow-Origin: *
 ```
 
+## Unexpected OPTIONS request
+
+If your application started to misbehave because of performing additional OPTIONS requests, it is most likely the issue with unwanted `sentry-trace` request header.
+This can happen when you are using too generic configuration for our Tracing Integration in the Browser SDK.
+In order to fix this, change `tracingOrigins` option during SDK intialization.
+For more details, see [Automatic Instrumentation](/platforms/javascript/performance/capturing/automatic/#tracingorigins) page of our Performance Monitoring docs.
```

Comment:
```suggestion
If your application started to misbehave because of performing additional OPTIONS requests, it is most likely an issue with unwanted `sentry-trace` request headers, which can happen when you are using too generic a configuration for our Tracing Integration in the Browser SDK.

To fix this, change the `tracingOrigins` option during SDK initialization. For more details, see [Automatic Instrumentation](/platforms/javascript/performance/capturing/automatic/#tracingorigins) in our Performance Monitoring documentation.
```